tests/foreman/cli/test_user.py

```diff
@@ -675,6 +674,33 @@ def test_negative_create_user_4(self, email):
         self.assertNotEqual(result.return_code, 0)
         self.assertTrue(result.stderr)
 
+    @skip_if_bug_open('bugzilla', 1204686)
+    def test_bugzilla_1204686(self):
+        """@Test: Create User with Empty Email Address
+
+        @Feature: User - Negative Create
+
+        @Steps:
+        1. Create User with empty Email Address in [2]
+        using valid Username, First Name, Surname, Language, authorized by
+
+        @Assert: User is not created. Appropriate error shown.
```

Comment:
How are you verifying that an appropriate error is shown? You can use `self.assertIn('desired error message', result.stdout)` to verify that some particular error message is shown.